### PR TITLE
enh: Allow to disable automatically open the wizard

### LINKED ISSUE #708: Don't show menu item if slides are disabled
When setting the app not to show any slides, the "About" menu item is still showing - but it's not doing anything.

If the app setting `slides` is empty, the menu item shouldn appear.

> The intro video slide is not cutomizable through that but can be disabled with removing it from the slide list with the following config:
> 
> 	occ config:app:set firstrunwizard slides --value="values,apps,clients,final"
> 
> _Originally posted by @juliushaertl in https://github.com/nextcloud/firstrunwizard/issues/470#issuecomment-770746778_

### PR DESCRIPTION
**Pull request by susnux**
* Supersedes #1106
* Resolves #708

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ A first run wizard that explains the usage of Nextcloud to new users
...
 ![](https://user-images.githubusercontent.com/3404133/51537050-bcc73e00-1e4d-11e9-8de0-29e6951c2b29.png)
 
+## Configuration
+
+No configuration is needed, but it is possible to prevent the wizard from opening for users by default.
+When disabled users can only open it from manually clicking "About" in the user menu.
+
+This can be done by setting an app setting value:
+
+```
```
Comment (szaimen) [inline README.md:14]: Actually the fiestunwizard app can be disabled so why should we need an additonall appconfig setting?

Comment (susnux) [reply README.md:14]: To not show the wizard all the time but only if manually requested by the user (clicking about).
Especially for the new changelog feature you might want to have it in the menu but not showing it all the time for new users.

Comment (szaimen) [reply README.md:14]: Ah I see, okay

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Comment by onny** [comment]
Is it possible to disable the wizard through global `config.php` file?

**Comment by susnux** [comment]
@onny either with that config implemented here you can disable it globally from opening by default. 
If you not even want it from being available as the "About" section of the user menu, then simply disable the app.

**Pull request merged**